### DISCUSSION TITLE
Varighed af statisk IPv6 prefix ved DHCPv6-PD opsætning

### DISCUSSION THREAD

**Discussion by netravnen** [emilstahl/ipv6#101] [General]
Hvad er code contributers erfaringer med

1. statiske, eller
2. næsten statiske

IPv6 præfikses når præfikses til lan-siden er tildelt med præfiks delegering (DHCPv6-PD)? 🤔

**Comment by netravnen** [comment]
Min egen opfattelse er de fleste udbydere har en lease tid.
Og hvis ens router et slukket i længere tid en udbyderens lease tid. Får min router et nyt IPv6 præfiks ved næste opstart.
Samme koncept vi kender fra IPv4 verdenen som "dynamisk offentlig ip".

**Reply by emilstahl** [reply]
Hov, jeg har slet ikke set den her. Det er mit indtryk er normen er at de assignes via DHCP, men er statiske?
